node/src/actors/chain_manager/mining.rs

```diff
@@ -369,7 +370,9 @@ fn build_block(
 
     // Include Mint Transaction by miner
     // TODO: Include Witnet's node PKH (keyed signature is not needed as there is no input)
```

Comment:
Good point, keyed signatures are only needed as proofs when we try to satisfy the requirements of a P2PKH challenge, i.e. a VTT output.